Add 1155 (#65)
mattpocock/ts-error-translator: packages/engine/errors/1155.md

---
original: "'const' declarations must be initialized."

> excerpt: "Const variables should be given a value straight away."

Reviewer: ```suggestion
excerpt: "A `const` must be given a value when it's declared."
```

---

I don't expect `const` variables to ever change after they're created. 
Therefore, I don't think it makes sense to create one without an initial value.

This confuses me:

```

> const name:string;

Reviewer: Could you fix the formatting here and change the formatting so it uses TS? I.e. use:

```ts

> ```
> const name:string;
> ```
> 
> But this is cool:
> 
> ```
> const name: string = "Jared";
> ```

Reviewer: ```suggestion
```ts
const name: string;
```

But this is cool:

```ts
const name: string = "Jared";
```
```
